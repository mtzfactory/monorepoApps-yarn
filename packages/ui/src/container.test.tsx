import { render, screen } from '@testing-library/react-native';

import Container from './container';

describe('Container', function () {
  it('Renders as expected', async function () {
    const testId = 'test-id';

    render(<Container testID={testId} />);

    const xxx = await screen.findByTestId(testId);

    expect(xxx).toBeDefined();

    expect(screen.toJSON()).toMatchSnapshot();
  });
});
