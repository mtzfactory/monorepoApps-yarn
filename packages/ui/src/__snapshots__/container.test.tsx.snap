// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Renders as expected 1`] = `
<View
  testID="test-id"
/>
`;
